replicating-fama-and-french-factors.qmd

```diff
@@ -21,24 +21,25 @@ tidy_finance <- dbConnect(
   extended_types = TRUE
 )
 
-crsp_monthly <- tbl(tidy_finance, "crsp_monthly") |>
-  collect()
-
-data_ff <- crsp_monthly |>
+data_ff <- tbl(tidy_finance, "crsp_monthly") |>
   select(
     permno, gvkey, month, ret_excess,
     mktcap, mktcap_lag, exchange
   ) |>
+  collect()
+
+data_ff <- data_ff |>
   drop_na()
 
-compustat <- tbl(tidy_finance, "compustat") |>
-  collect()
+book_equity <- tbl(tidy_finance, "compustat") |>
+    select(gvkey, datadate, be) |>
+    collect()
 
-be <- compustat |>
-  select(gvkey, datadate, be) |>
+book_equity <- book_equity |>
   drop_na()
```

Comment:
Why not add the drop_na() to the code block above before the collect() (or after if there is no sql translation yet for it)?

Reply:
Done in [7efc916](https://github.com/tidy-finance/website/pull/42/commits/7efc916a7886f61f7665d582c4e349817f979ed2)